multi-tenancy-observability/README.md

```diff
@@ -0,0 +1,154 @@
+---
+creation_date: 2024-02-08
+issues:
+- https://github.com/giantswarm/roadmap/issues/2771
+owners:
+- https://github.com/orgs/giantswarm/teams/team-atlas
+state: review
+summary: How Giant Swarm manages multi-tenancy to isolate observability data per tenants (i.e. metrics, logs, traces).
```

Comment:
I stumbled over _this PR_ from `#noise-rfc` and because of the summary. Since I'm currently preparing some best practices for technical writing, I have some nits regarding technical writing quality 😉. I know the RFC is still being drafted, but I wanted to leave these comments already.

---

This is not a summary, but the title of the RFC. Please put here what the RFC proposes, i.e. actions and changes.

Reply:
Could you maybe give me examples so I could improve?